webcurator-store/src/main/resources/application.properties

```diff
@@ -58,7 +58,7 @@ webapp.baseUrl=http://localhost:8080/wct
 #WaybackIndexer
 
 # Enable this indexer
-waybackIndexer.enabled=true
+waybackIndexer.enabled=false
```

Comment:
Is this change of default necessary? It could lead to some admins being unpleasantly surprised after the upgrade.